src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <section class="calendar calendar--29">
+      <div class="calendar__date calendar__date--1 calendar__date--Sunday"></div>
```

Comment:
Replace the hardcoded day modifier (e.g., calendar__date--1) with an automated solution using ::before and @for.